src/components/view.js

```diff
@@ -49,6 +51,7 @@ export default {
           matched.instances[name] = undefined
         }
       }
+      data.props = resolveProps(route, component, matched.props && matched.props[name])
```

Comment:
The props of <router-view> are still passed to the rendered element. The <div> in Hello still gets the `class="view"`